pages/accountLists/[accountListId]/tools/useToolsHelper.ts

```diff
@@ -3,16 +3,21 @@ import { useCallback } from 'react';
 import { useAccountListId } from 'src/hooks/useAccountListId';
 import { getQueryParam } from 'src/utils/queryParam';
 
-export type SetContactFocus = (contactId: string) => void;
+export type SetContactFocus = (contactId: string, tabKey?: string) => void;
 
 export const useToolsHelper = () => {
   const { query, push } = useRouter();
   const accountListId = useAccountListId();
   const selectedContactId = getQueryParam(query, 'contactId');
 
   const handleSelectContact = useCallback(
-    (pagePath: string, contactId: string) => {
-      push(`/accountLists/${accountListId}/${pagePath}/${contactId}`);
+    (pagePath: string, contactId: string, tabKey?: string) => {
+      tabKey
+        ? push({
+            pathname: `/accountLists/${accountListId}/${pagePath}/${contactId}`,
+            query: { tabKey: tabKey },
+          })
+        : push(`/accountLists/${accountListId}/${pagePath}/${contactId}`);
```

Comment:
Just to remove repeating code. Otherwise this is great.
```suggestion
    (pagePath: string, contactId: string, tabKey?: string) => {
      const pathname = `/accountLists/${accountListId}/${pagePath}/${contactId}`
      tabKey
        ? push({
            pathname,
            query: { tabKey: tabKey },
          })
        : push(pathname);
```